### ISSUE TITLE
bug(mat-stepper): failing basic accessibility rule

### ISSUE THREAD

**Issue by jennifergoncalves** [angular/components#26507]
### Is this a regression?

- [ ] Yes, this behavior used to work in the previous version

### Description

mat-stepper is creating the tabpanels INSIDE the tablist. This is semantically incorrect and fails an accessibility rule surrounding roles.  The "tabs" should be in the tablist, not the tabpanels.

<img width="1599" alt="Screenshot 2023-01-25 at 11 38 59 AM" src="https://user-images.githubusercontent.com/307145/214857735-baa6d768-c9ae-45e4-bcaa-e7ec121176fd.png">
<img width="1638" alt="Screenshot 2023-01-25 at 12 00 25 PM" src="https://user-images.githubusercontent.com/307145/214857739-9f53723c-7211-4208-b73b-a74c4d727aad.png">

### Reproduction

Steps to reproduce:
1. run accessibility testing (I use the Chrome AXE plugin) on the documentation site: https://material.angular.io/components/stepper/overview

### Expected Behavior

The "tabs" should be in the tablist, the tabpanels should NOT be inside the tablist

### Actual Behavior

mat-stepper is creating the tabpanels INSIDE the tablist. 

### Environment

- Angular:  https://material.angular.io/components/stepper/overview
- CDK/Material: 15.1.2 (tested back to 12 and that fails as well)
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): MacOS

**Comment by crisbeto** [comment]
Duplicate of https://github.com/angular/components/issues/26444. For what it's worth, we do accessibility testing and this doesn't affect actual users.
